share/spice/piratetalk/piratetalk.js

```diff
@@ -0,0 +1,28 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_piratetalk = function(api_result) {
+
+        if (!api_result || api_result.translation==="") {
+            return Spice.failed('piratetalk');
+        }
+        
+        Spice.add({
+            id: "piratetalk",
+            name: "Answer",
+            data: {
+                title:api_result.translation,
+            },
+            meta: {
+                sourceName: "Pirate",
+                sourceUrl: 'https://pirate-speak.herokuapp.com/',
+            },
+            templates: {     
+                group: 'base',
```

Comment:
@thejasbabu We should be able to simplify things by removing the handlebars and CSS - as @moollaza it's preferable to stick to an existing template (it stresses this in the docs), you can do minor tweaking (change font size, color etc.) in the CSS *without* needing to change template.